homeassistant/components/matter/switch.py

```diff
@@ -58,6 +67,66 @@ def _update_from_device(self) -> None:
         )
 
 
+class MatterGenericCommandSwitch(MatterEntity, SwitchEntity):
```

Comment:
```suggestion
class MatterGenericCommandSwitch(MatterSwitch):
```

Reply:
I think we should drop the MatterSwitch and adjust OnOff cluster schema to your new proposal which can then carry the name MatterSwitch. Now things feel a bit duplicated.